ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellDemoController.swift

```diff
@@ -11,14 +11,6 @@ import UIKit
 class TableViewCellDemoController: DemoTableViewController {
     let sections: [TableViewSampleData.Section] = TableViewCellSampleData.sections
 
-    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
```

Comment:
why are these deleted?

Reply:
They were removed so that the tableview defaults to its insetGroup style set in DemoTableViewController. See #1535